scheduler/docs/index.rst

```diff
@@ -5,5 +5,7 @@ Api Reference
 .. toctree::
     :maxdepth: 2
 
+    gapic/v1/api
+    gapic/v1types
```

Comment:
Should we be separating the older / beta versions with language like that in other APIs, e.g. from `vision`:

```rst
Previous beta releases ``v1p3beta1`` and ``v1p2beta1`` are provided as well.

An API and type reference is provided for ``v1p3beta1``:

.. toctree::
    :maxdepth: 2

    gapic/v1p3beta1/api
    gapic/v1p3beta1/types


An API and type reference is provided for ``v1p2beta1``:

.. toctree::
    :maxdepth: 2

    gapic/v1p2beta1/api
    gapic/v1p2beta1/types
```

Reply:
Done